packages/mendel-development/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mendel-development",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
`lerna` would have all this options. Just following old ways for now.
I don't like bumping all for a number of reasons, I tried that in the past and it is quite awkward, specially when you have lots of plugins that have different speeds of development. We can adopt this for all of core packages. It would be quite nice. But than it becomes a good idea to bake the changes in master before releasing a bit more slowlly